charts/web-wallet/values.yaml

```diff
@@ -5,18 +5,27 @@ image:
 mobileLayoutImage:
   repository: us.gcr.io/galoy-org/web-wallet-mobile-layout
   digest: "sha256:40e4edbd98cb3d58bda687265fbf0230c6c80745e69778db15218c287ee0e18f"
-useMobileLayout: false
```

Comment:
Shouldn't this be kept at top level? This is only used by helm

Reply:
I contemplated that. The thing is I don't like it as a lone top level config (everything else is nested) and I don't know a better key to scope it under.

Reply:
Perhaps we can have a `mobileLayout` top level field:

```
mobileLayout:
  enabled: boolean
  image:
    repo: string
    digest: string  
```